### DISCUSSION TITLE
Dynamic Topic Name

### DISCUSSION THREAD

**Discussion by m2b** [thomassorensen2650/node-red-contrib-mqtt-sparkplug-plus#45] [General]
I am not sure if this is supported but in many cases I have multiple devices on the same edge node.  Would be good if the msg send includes the topic and not just the metrics.  Else, I am having to configure multiple Sparkplug out nodes.  
![image](https://github.com/thomassorensen2650/node-red-contrib-mqtt-sparkplug-plus/assets/773233/dd2d3374-df64-4f53-81c0-fcc7ad95c2b9)

**Comment by thomassorensen2650** [comment]
you can do that. if you leave the topic empty in the Sparkplug Out node, then you can set the topic using msg.topic
